<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionProduitDialog</class>
 <widget class="QDialog" name="GestionProduitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajouter / supprimer un produit</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_detail_produit">
     <property name="font">
      <font>
       <family>Outfit</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {border: none; border-radius: 4px;}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_nom_produit">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom du produit</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="input_nom_produit">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border: 1px solid rgb(180, 180, 180);
border-radius: 4px;
padding: 2px 5px;
}

QLineEdit:focus {
border: 1px solid rgb(85, 170, 255);
}</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_unite_base">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Unité de base</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="input_unite_base">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
border: 1px solid rgb(180, 180, 180);
border-radius: 4px;
padding: 2px 5px;
}

QLineEdit:focus {
border: 1px solid rgb(85, 170, 255);
}</string>
        </property>
        <property name="placeholderText">
         <string>exemple: kg, kapoka, sac</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_stock_initial">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stock initial (en unité de base):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spin_box_stock_initial">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Only applicable when adding a new product. For existing products, use 'Add Stock' feature.</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QDoubleSpinBox {
    border: 1px solid #CCCCCC;
    border-radius: 3px;
    padding: 2px 5px;
}
QDoubleSpinBox::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;
    width: 16px;
    border-left: 1px solid #CCCCCC;
   background: #cccccc;
}
QDoubleSpinBox::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;
    width: 16px;
    border-left: 1px solid #CCCCCC;
    background: #cccccc;
}
QDoubleSpinBox::up-arrow {
    image: url(:/icons/icons/caret-up.svg);
    width: 12px;
    height: 12px;
}
QDoubleSpinBox::down-arrow {
    image: url(:/icons/icons/caret-down.svg);
    width: 12px;
    height: 12px;
}
QDoubleSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background: #66BB6A;
}
QDoubleSpinBox::up-button:pressed, QSpinBox::down-button:pressed {
    background: #388E3C; 
}</string>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>999999999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_unite_vente">
     <property name="font">
      <font>
       <family>Outfit</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Unité de vente</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="table_view_unite_vente">
        <property name="font">
         <font>
          <family>Outfit</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QHeaderView::section:horizontal {
	background-color: rgb(122, 194, 145); padding: 4px; border: none; border-left: 0.5px solid rgb(65, 103, 70); color: rgb(246, 255, 249)

}</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>115</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="font">
           <font>
            <family>Outfit</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_ajouter_unite">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
background-color:  rgba(87, 182, 255, 50);
 padding: 5px 10px;
 border: 2px solid rgb(0, 85, 255);
border-radius: 4px;
color: rgb(0, 85, 255); 
font: bold;
}

QPushButton:hover {
	background-color:  rgba(87, 182, 255, 80) ; 
}</string>
          </property>
          <property name="text">
           <string>Ajouter un unité</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_modifier_unite">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
background-color:  rgba(87, 182, 255, 50);
 padding: 5px 10px;
 border: 2px solid rgb(0, 85, 255);
border-radius: 4px;
color: rgb(0, 85, 255); 
font: bold;
}

QPushButton:hover {
	background-color:  rgba(87, 182, 255, 80) ; 
}</string>
          </property>
          <property name="text">
           <string>Modifier l'unité</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bouton_supprimer_unite">
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">
QPushButton {
background-color:  rgba(255, 152, 152, 50);
 padding: 5px 10px;
 border: 2px solid rgb(255, 64, 64);
border-radius: 4px;
color:  rgb(255, 64, 64); 
font: bold;
}

QPushButton:hover {
	background-color:   rgba(255, 152, 152, 80) ; 
}</string>
          </property>
          <property name="text">
           <string>Supprimer l'unité</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_annuler">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton {
background-color:  rgba(217, 217, 217, 100);
 padding: 5px 10px;
 border: 2px solid rgb(102, 102, 102);
border-radius: 4px;
color:   rgb(102, 102, 102); 
font: bold;
}

QPushButton:hover {
	background-color:    rgba(217, 217, 217, 150) ; 
}</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_enregistrer">
       <property name="font">
        <font>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">
QPushButton {
background-color:  rgba(87, 182, 255, 50);
 padding: 5px 10px;
 border: 2px solid rgb(0, 85, 255);
border-radius: 4px;
color: rgb(0, 85, 255); 
font: bold;
}

QPushButton:hover {
	background-color:  rgba(87, 182, 255, 80) ; 
}</string>
       </property>
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
